code/modules/clothing/clothing.dm

```diff
@@ -22,6 +22,8 @@
 	..()
 	
 /obj/item/clothing/CtrlClick(var/mob/user)
+	if(istype(loc,/turf))
+	return ..()
```

Comment:
You have to indent this!

Reply:
What's wrong exactly, this is how i indented it locally when i tested it, everything else i tried said something about inconsistent indentation

Reply:
okay I think I might see what I need to do looking at other code.